libs/blocks/youtube/youtube.js

```diff
@@ -14,12 +14,18 @@ export default function init(a) {
         webkitallowfullscreen mozallowfullscreen allowfullscreen
         allow="encrypted-media; accelerometer; gyroscope; picture-in-picture"
         scrolling="no"
+        id="player-${id}"
         title="${title}">
       </iframe>
     </div>`;
     a.insertAdjacentHTML('afterend', embedHTML);
     a.remove();
+    document.addEventListener('readystatechange', () => {
+      if (document.readyState === 'complete') {
```

Comment:
Does it need the `readystatechange` complete event listener? I would be under the impression the page is already ready and analytics have been added by the time we render this block

Reply:
@mokimo : Sometimes Launch's load is not completed and it fails to trigger the call. So to make sure that everything is loaded, this EventListener is added.